lighthouse-core/test/audits/cache-start-url.js

```diff
@@ -30,50 +30,51 @@ describe('Cache: start_url audit', () => {
   });
 
   it('fails when no cache contents given', () => {
-    return assert.equal(Audit.audit({Manifest, URL}).rawValue, false);
+    const artifacts = {Manifest: exampleManifest, URL};
+    const output = Audit.audit(artifacts);
+    assert.equal(output.rawValue, false);
+    assert.equal(output.debugString, 'No cache or URL detected');
   });
 
   it('fails when no URL given', () => {
-    return assert.equal(Audit.audit({Manifest, CacheContents}).rawValue, false);
+    const artifacts = {Manifest: exampleManifest, CacheContents};
+    const output = Audit.audit(artifacts);
+    assert.equal(output.rawValue, false);
+    assert.equal(output.debugString, 'No cache or URL detected');
   });
 
   // Need to disable camelcase check for dealing with start_url.
   /* eslint-disable camelcase */
   it('fails when a manifest artifact contains no start_url', () => {
-    const inputs = {
+    const artifacts = {
```

Comment:
maybe just delete this test since it's basically covered by the one you added below